.github/workflows/ci.yml

```
@@ -47,3 +50,33 @@ jobs:
     - name: Run tests
       run: |
         uv run -m unittest test.rag
+    
+    - name: Set up environment
+      run: |
+        cp .env.ci .env
```

In case you move `.env.ci` into `.github/workflows`:
```suggestion
        cp .github/workflows/.env.ci .env
```